Assets/_BForBoss/_Utility/Scripts/Editor/PropertyDrawers/ResolverPropertyDrawer.cs

```diff
@@ -153,21 +153,24 @@ private ComponentInformation[] GetResolvedGameObjects(MonoBehaviour mb, ResolveT
             {
                 case ResolveType.FromChildren:
                 {
-                    foreach (Transform child in mb.transform)
+                    Transform[] transforms = mb.GetComponentsInChildren<Transform>(_includeInactiveGameObjects)
```

Comment:
What are these changes for?

Reply:
A
    - B
     --C [Resolve]
       --- D
          ----H
       --- E
          ----F
          ----G


So does it try to find all of D,H,E,F,G? 
Where as before it goes for only D, H?

Reply:
Basically I realised that the way I had before did not account for children of children (which is how GetComponentsInChildren works) so it was inconsistent with the functionality when trying to resolve components instead of GameObjects. 

So in this case it would find D,H,E,F,G (like how the component one would work). This way also allows the `IncludeInactive` parameter to take into account for the gameObject search as well